modules/home-manager/options/misc.nix

```diff
@@ -48,5 +48,10 @@ in {
         with the canvas (practically anything at this point)
       '';
     };
+
+    firefoxSync = mkEnableOption ''
+      Enable Firefox Sync. Disabled by default to prevent calling home.
+      Quite sane to want it re-enabled as it's encrypted and convenient.
```

Comment:
Remove this line. If you'd like to mention it's encrypted, leave a source to that.

Reply:
so would you prefer me to just leave "Firefox Sync" in the desc?

Reply:
yes, that would work.